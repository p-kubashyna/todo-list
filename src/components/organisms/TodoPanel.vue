<script setup>
import TodoList from '@/components/organisms/TodoList.vue'
import TodoBottomPanel from '@/components/molecules/TodoBottomPanel.vue'
</script>

<template>
<div class="test">
  <TodoList/>
  <TodoBottomPanel/>
</div>
</template>

<style>
.test {
  margin-top: 30px;
  width: 600px;
  border-radius: 7px;
  border-width: 5px;
  border-color: white;
  border-style: solid;
  box-shadow: -15px 15px 30px 1px rgba(0, 0, 0, .25);
}
</style>
