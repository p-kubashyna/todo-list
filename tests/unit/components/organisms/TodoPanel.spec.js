import { describe, expect, it } from "@jest/globals";
import { shallowMount } from "@vue/test-utils";
import TodoPanel from "@/components/organisms/TodoPanel.vue";

describe("Given the TodoPanel component", () => {
  it("should render", () => {
    const wrapper = shallowMount(TodoPanel, {});

    expect(wrapper.element).toMatchSnapshot();
  });
});
