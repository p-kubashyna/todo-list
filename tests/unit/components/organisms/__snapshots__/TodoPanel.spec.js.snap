// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the TodoPanel component should render 1`] = `
<div
  class="panel"
>
  <todo-list-stub />
  <todo-bottom-panel-stub />
</div>
`;
